rust/src/protocol_types/governance/drep.rs

```diff
@@ -109,23 +205,36 @@ impl DRep {
         bech32::encode(&hrp, data.to_base32()).map_err(|e| JsError::from_str(&format! {"{:?}", e}))
     }
 
+    pub fn to_bech32(&self) -> Result<String, JsError> {
+        self.internal_to_bech32("drep".to_string())
+    }
+
+    pub fn to_bech32_cip129(&self) -> Result<String, JsError> {
```

Comment:
probably we need to encode only in cip129 format and do not provide API to encode in a old format when cip129 will be merged